setup.py

```diff
@@ -41,9 +40,14 @@
     packages=find_packages("src"),
     package_dir={"": "src"},
     license="BSD",
-    long_description=open("README.md").read(),
+    long_description=open("README.md")
```

Comment:
please add a comment referencing this issue and the pypi issue so that we could remove the hack again once pypi fixes there stuff